PowerEditor/src/ScitillaComponent/FindReplaceDlg.cpp

```diff
@@ -2764,6 +2769,13 @@ void FindReplaceDlg::execSavedCommand(int cmd, uptr_t intValue, const generic_st
 						break;
 					}
 
+					case IDC_CLEAR_ALL:
+					{
+						(*_ppEditView)->clearIndicator(SCE_UNIVERSAL_FOUND_STYLE);

```

Comment:
@sasumner:  I am probably wrong on this, but are you sure this line is not supposed to be `SCI_UNIVERSAL_FOUND_STYLE`?